### DISCUSSION TITLE
Route to the next petrol station

### DISCUSSION THREAD

**Discussion by HannesHoBu** [devemux86/cruiser#153] [Q&A] [answered]
I haven't seen a discussion on this topic.
May it be worth to consider the possibility to drive to the next petrol station in case you are short of gasoline?
I would appreciate a button showing the petrol stations in the vicinity.
This is one of the few things I like using Cali....

**Comment by 0709wiwiwi** [comment]
Similar.
https://github.com/devemux86/cruiser/discussions/9

**Comment by devemux86** [comment, marked as answer]
Hi @HannesHoBu,

It is already implemented! :slightly_smiling_face: 

Please see this discussion:
- https://github.com/devemux86/cruiser/discussions/9
